modules/govuk_jenkins/templates/jobs/check_content_consistency.yaml.erb

```diff
@@ -0,0 +1,87 @@
+---
+- scm:
+    name: check-content-consistency
+    scm:
+      - git:
+          url: git@github.gds:gds/router-data.git
+          branches:
+            - master
+- job:
+    name: Check_Content_Consistency
+    display-name: Check_Content_Consistency
+    project-type: freestyle
+    description: |
+      This job checks for consistency between the router-api, content-store
+      and publishing-api.
+    scm:
+      - check-content-consistency
+    logrotate:
+      numToKeep: 10
+    triggers:
+        - timed: 'H 5 * * *'
+    builders:
+      - shell: |
+          #!/bin/bash
+          set -eu
+
+          cd "${WORKSPACE}"
+
+          function run_on_router_backend() {
+            ssh deploy@router-backend-1.backend.<%= @app_domain -%> "$@"
```

Comment:
if you're using `@app_domain` this will need to be added as a parameter to the manifest